### PR TITLE
Addding some awesomeness

### PR DESCRIPTION
**Pull request by afiune**
Awesome comments

### CODE REVIEW

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

File: recipes/default.rb [hunk 1]
```diff
@@ -3,3 +3,6 @@
 # Recipe:: default
 #
 # Copyright (c) 2015 The Authors, All Rights Reserved.
+
+log "This is a nice resource"
```
Comment (afiune) [inline recipes/default.rb:7]: This is actually a great resource

**Comment by afiune** [comment]
@delivery approve

**Comment by chef-delivery** [comment]
Change: 84472357-20f1-4c70-a311-5b569eb14284 approved by: @afiune

**Comment by chef-delivery** [comment]
This PR has passed 'Acceptance' and is ready to be delivered!
  Use: '@delivery deliver' when validated in acceptance.

**Pull request merged**